addons/crm/models/crm_lead.py

```diff
@@ -1432,7 +1432,7 @@ def _pls_get_naive_bayes_probabilities(self, batch_mode=False):
                     s_lead_lost *= value_result['lost'] / total_lost
 
             # 3. Compute Probability to win
-            lead_probabilities[lead_id] = 100 * s_lead_won / (s_lead_won + s_lead_lost)
+            lead_probabilities[lead_id] = round(100 * s_lead_won / (s_lead_won + s_lead_lost), 2)
```

Comment:
Note that we could also round to 0 decimals.
This would mean max 100 transactions instead of 10.000.
And I guess User doesn't care if we display 46% instead 46,23%
To discuss with the decision guys maybe !